docs/development/benchmark_with_djl.md

```diff
@@ -0,0 +1,217 @@
+# Benchmark your DL model with DJL
+
+DJL offers a comprehensive script to benchmark the model on all different 
+platforms for single-thread/multi-thread inference performance.This document will guide you how to run benchmark with DJL.
+
+## Prerequisite
```

Comment:
We should write the prerequisites starting from cloning DJL. Otherwise, users who are less familiar may be confused with how to start.